_drafts/rails-new-for-you.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: A "rails new" For You
+---
+
+Recently, I went on a personal endeavor to determine the best way to create
+a new Rails app pre-configured to my liking. I was sick and tired of wasting
+time replacing and configuring test runners, adding development gems I prefer,
+and reinventing the README. Why did it take so long for me to do this?
+
+## .railsrc
+
+For some reason it took me a long time to discover this handy script. Following
+the convention of other ["run configuration" files](wikipedia-config-files),
+you can add options here that you wish to be included every time you `rails
+new`.
+
+My config...
+
+* skips `bundle install` (handled later)
+* skips `.keep` files
+* skips [spring](https://github.com/rails/spring)
+* skips Test::Unit (I prefer RSpec)
+* users PostgreSQL instead of SQlite
+
+Here it is:
+
+<script src="http://gist-it.appspot.com/https://github.com/iamvery/dotfiles/blob/c9e10af36a17d4b21b3dcb4dedfcd08acc7af280/.railsrc"></script>
+
+## Rails App Template
+
+The command line options can only get me so far. To complete my desired
+pre-configuration, it's time to automate the things with [Rails application templates](rails-app-templates).
+
+App templates allow you to define a procedure used to config an app. It has
+built in hooks for `bundler` and `git`. My config...
+
+* installs preferred gems
+* configures test runner
+* removes turbolinks
```

Comment:
I actually looked for it and couldn't find it online, it's also not in `rails new --help`... maybe it _isn't_ a thing? Will have to test